src/helpers.ts

```diff
@@ -150,31 +154,33 @@ export const request = async <T = Response>(
       ...(options ?? {}),
     }
 
-    if (userId) {
-      updatedOptions.headers = {
-        ...options?.headers,
-        'X-LIFI-UserId': userId,
+    if (!skipTrackingHeaders) {
```

Comment:
Adding this because tracking headers were going to npm registry as well and causing the request to fail.